src/alchemtest/gomc/benzene/descr.rst

```diff
@@ -0,0 +1,32 @@
+GOMC: Benzene in water
+=========================
+
+Hydration free energy of benzene using `TraPPE-EH <https://pubs.acs.org/doi/10.1021/jp073586l>`_ 
+model and SPC water model.
+
+Notes
+-----
+Data Set Characteristics:
+    :Number of Legs: 2 (Coulomb, VDW)
+    :Number of Windows: 7 for Coulomb, 15 for VDW
+    :Length of Windows: 50 million Monte Carlo steps
+    :System Size: 1001 molecules
+    :Temperature: 298 K
+    :Pressure: 1 bar
```

Comment:
Please also add the units for energy and time, similar to https://github.com/alchemistry/alchemtest/blob/master/src/alchemtest/gmx/benzene/descr.rst (was recently added in PR #32 ).

Reply:
And yes, the unit of time can just be "MC step".